README.md

```diff
@@ -54,8 +58,14 @@ This is the default web application built on top of the MifosX platform for the
     Waiting...Warning: EMFILE, too many open files
 
    ```
+   
+1. You can use these credentials to log in
 
-1. Default username/password: mifos/password. This application will hit the demo server by default.
+   ```
+    Username: mifos
+    Password: password
+   ```
+This application will hit the demo server by default.
```

Comment:
Keep this line at the end as a note.

Reply:
the end of the README file or the "Building from source" section ?